iesurveychecks/1-adapt-scto-forms.md

```diff
@@ -0,0 +1,28 @@
+
+# 1. Adapting SurveyCTO forms
+When [programming a questionnaire for electronic data collection](https://docs.surveycto.com/02-designing-forms/01-core-concepts/), it requires to be adapted to include the data quality checks by taking into account all possible scenarios that an enumerator may encounter during the collection process. (Instructions on how to create a SurveyCTO form can be found [here). When designing your survey form, it is important to take into account all possible scenarios that an enumerator may encounter during the collection process.
```

Comment:
link to scto form creation instructions not working. last sentence is redundant, suggest deleting.